From 1dc2b181d5e44e7ce15a7aa6a72331c530ac5e48 Mon Sep 17 00:00:00 2001
From: Jason <r64343@freescale.com>
Date: Thu, 30 Oct 2008 16:57:36 +0800
Subject: [PATCH] ENGR00096858 MX35: Kernel can't bootup randomly

Kernel can't bootup randomly, which was caused by
set NFC_RST bit at init phase.Remove it from init
phase.

Signed-off-by:Jason Liu <r64343@freescale.com>
---
 drivers/mtd/nand/mxc_nd2.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/drivers/mtd/nand/mxc_nd2.c b/drivers/mtd/nand/mxc_nd2.c
index 5a8e82c..f77ee3b 100644
--- a/drivers/mtd/nand/mxc_nd2.c
+++ b/drivers/mtd/nand/mxc_nd2.c
@@ -1100,9 +1100,6 @@ static int mxc_nand_scan_bbt(struct mtd_info *mtd)
 
 static void mxc_nfc_init(void)
 {
-	/* Resetting  NFC */
-	raw_write((raw_read(REG_NFC_RST) | NFC_RST), REG_NFC_RST);
-
 	/* Disable interrupt */
 	raw_write((raw_read(REG_NFC_INTRRUPT) | NFC_INT_MSK), REG_NFC_INTRRUPT);
 
-- 
1.5.4.4

